### DISCUSSION TITLE
How to import css？

### DISCUSSION THREAD

**Discussion by sylvoslee** [idom-team/idom#434] [Question] [answered]
How to import css？

**Comment by rmorshea** [comment, marked as answer]
There's several answers to this:

1. Use a [`link`](https://idom-docs.herokuapp.com/docs/auto/api-reference.html#idom.html.link) element
2. Write your own client for IDOM that imports your CSS
3. There's also an [open issue](https://github.com/idom-team/idom/issues/253) about being able to customize the built-in client
